test/shmemx/Makefile.am

```diff
@@ -20,13 +20,16 @@ endif
 
 if SHMEMX_TESTS
 check_PROGRAMS += \
-	c11_test_shmem_ibput \
-	c11_test_shmem_ibget \
-	perf_counter \
-	shmemx_team_node \
-	shmem_malloc_with_hints \
-        signal_add \
-        signal_set
+    c11_test_shmem_ibput \
+    c11_test_shmem_ibget \
+    perf_counter \
+    shmemx_team_node \
+    shmem_malloc_with_hints \
+    signal_add \
+    signal_set
+    ex_scan \
+    in_scan \
+    scan_in_place
```

Comment:
Only the last 4 lines should have changes here. The indentation need not be changed.

Reply:
I changed them because they were tabs and the rest of the files did not have tabs. I thought it would make it consistent. I can change it back to tabs if you think it is better. Which do you prefer?

Reply:
No preference. Just wanted to make sure that they are all unified in terms of indentation.